src/cmake-tools.ts

```diff
@@ -2182,7 +2182,9 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
         }
 
         const quoted = shlex.quote(executable.path);
-        this._launchTerminal.sendText(`${quoted} ${launchArgs}`);
+        const quotedCommand = `${quoted} ${launchArgs}\n`;
+        console.log(`execute command:${quotedCommand}`);
```

Comment:
Maybe also log the `cwd`.  My suspicion is that the file is not being written in the location we expect it to be.  We could also consider doing a recursive search for the file from a higher up directory.